### ISSUE TITLE
the loss suddenly increases

### ISSUE THREAD

**Issue by yykk1xx** [microsoft/DCVC#66]
Why does the loss suddenly increase by several times after three to four hundred iterations during model training? Another question is, during each iteration of training, does the function net.update() need to run once (with net = DMC())?

**Comment by RXGuo** [comment]
I ran into a similar problem when training on volumetric images. After a certain number of iterations the loss suddenly increases rapidly followed by overflow.

**Comment by goldbama** [comment]
> Why does the loss suddenly increase by several times after three to four hundred iterations during model training? Another question is, during each iteration of training, does the function net.update() need to run once (with net = DMC())?
Do you have the training code for DCVC？
